packages/strapi-plugin-email/admin/src/index.js

```diff
@@ -30,6 +35,28 @@ export default strapi => {
     pluginLogo,
     preventComponentRendering: false,
     trads,
+    settings: {
+      menuSection: {
+        id: pluginId,
+        title: getTrad('SettingsNav.section-label'),
+        links: [
+          {
+            title: {
+              id: getTrad('SettingsNav.link.settings'),
+              defaultMessage: 'Settings',
+            },
+            name: 'settings',
+            to: `${strapi.settingsBaseURL}/${pluginId}`,
+            Component: () => (
+              <CheckPagePermissions permissions={pluginPermissions.settings}>
```

Comment:
Since we cannot edit the permissions I am not sure if they should be applied. Also according to permissions file the `settings` key does not exist.